snooty/rstspec.toml

```diff
@@ -1564,3 +1563,106 @@ prefix = "opsmgrkube"
 [rstobject."mongodb:autoencryptkeyword"]
 help = "Automatic Client-Side Field Level Encrytion Rules Keyword"
 prefix = "autoencryptkeyword"
+
+[tabs]
+cloudproviders = [
```

Comment:
I believe this key should be `cloud-providers` to match the [legacy implementation](https://github.com/mongodb/docs-tools/blob/master/sphinxext/tabs.py#L399-L405). As is, the parser is unable to match `tabs-cloud-providers` tabsets to this spec definition.